### ISSUE TITLE
different number of labels of objects, relationships and attributes from the paper

### ISSUE THREAD

**Issue by wtliao** [husthuaan/AoANet#28]
Hi, thanks for sharing the nice idea and code. I am trying to extract scene graph from the VG dataset using motifs code. However, as you mentioned in the paper that **305 objects, 103 attributes and 64 relationships are kept by filtering the objects,  attributes and  relationships which apear more than 2000 times in the trianing set.** I am trying to get these labels from the raw dataset using the [data split](https://github.com/danfeiX/scene-graph-TF-release/blob/master/data_tools/README.md). But finally I get 226 objects, 72 attributes and 39 relationships from the training set and 329 objects, 97 attributes and 60 relationships from the whole dataset. So could you share your label list of object, relationships and attributes? Thanks in advanced.
